scripts/prepare_automatic_backups.py

```diff
@@ -44,6 +54,15 @@ def generate_backup_url(cloud_storage_bucket_name, module_class_names):
 
 
 def update_cron_dict(cron_dict):
+    """Adding new urls in dict for backup.
```

Comment:
Adding --> Adds. Make similar changes below.

Ensure all docstrings have consistent grammar with one another. I mentioned this in an earlier review comment.

Reply:
sorry , next time will be taken care.
done it.